code/code/sys/configuration.h

```diff
@@ -1,7 +1,15 @@
 #pragma once
 
+#include <string>
+#include <string_view>
```

Comment:
Propagating this include into 70 code files is hurting compilation times, even if just a bit.

Reply:
Yeah, I think in retrospect there's probably no reason to worry about writing functions that could be used in the future like this, and instead I should just write stuff to be as efficient as possible in the context at hand. I'll combine these functions and move them into an anonymous namespace in `main.cc`.

Reply:
Fixed in https://github.com/sneezymud/sneezymud/pull/476/commits/17de1b40cc65e5f489f434878ab0267af991d881